lib/reline/ansi.rb

```diff
@@ -7,6 +7,10 @@ class Reline::ANSI
   CAPNAME_KEY_BINDINGS = {
     'khome' => :ed_move_to_beg,
     'kend'  => :ed_move_to_end,
+    'kich1' => :ed_ignore,
+    'kdch1' => :key_delete,
+    'kpp' => :ed_ignore,
+    'knp' => :ed_ignore,
```

Comment:
@sshock Why do you assign ed_ignore to kinch1, kpp, knp?

Reply:
Hi @ima1zumi.

It fixes it so Insert, PgUp, and PgDn will no longer spew out ugly escape codes like `^[[2~`, `^[[5~` and `^[[6~`.

Instead hitting any of these keys will just do nothing.

Reply:
I can take these three out and put them in a separate MR if needed.

Reply:
Thank you for your response!
I think we should get kich1, kpp, and knp working correctly.
I believe the following behavior is correct.

- Insert key is in overwrite mode
- PGUP, PGDN keys work the same way as the arrow keys

`ed_ignore` is not the expected behavior.  I suggest that kinch1, kpp, and knp excludes in this PR.

> I can take these three out and put them in a separate MR if needed.

That would be great!

Reply:
@ima1zumi these three have now been removed from this MR